packages/gatsby-source-shopify/src/create-client.js

```diff
@@ -2,9 +2,17 @@ import { GraphQLClient } from "graphql-request"
 /**
  * Create a Shopify Storefront GraphQL client for the provided name and token.
  */
-export const createClient = (shopName, accessToken) =>
-  new GraphQLClient(`https://${shopName}.myshopify.com/api/graphql`, {
+export const createClient = (shopName, accessToken) => {
+  let url;
+  if (shopName.indexOf("." > -1)) {
```

Comment:
Looks like you may not have tested this because that looks like a typo.
```suggestion
  if (shopName.indexOf(".") > -1) {
```

Additionally it should use `.includes` for this:

```suggestion
  if (shopName.includes(".")) {
```

And it should run the styles, so the post-commit / prettier hook somehow didn't run. But we can fix that.

Reply:
I manually tested that a domain works, but with the typo, the condition is always true!